packs/data/extinction-curse-bestiary.db/doblagub.json

```diff
@@ -928,7 +1010,7 @@
                 },
                 "slug": null,
                 "source": {
-                    "value": ""
+                    "value": "https://2e.aonprd.com/Spells.aspx?ID=172"
```

Comment:
Replace with compendium spells. We shouldn't have any AoN links in sources anymore

Reply:
I was trying to avoid looking at this one much, since it's the book that I'm in. The whole AP needs some review.